recipes/ipyfilechooser/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ipyfilechooser" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 705a8a0d36b87414a72ab81a0327d3201a85e8f68ba1c70aa4f0250e1192a0db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - ipywidgets
+    - pip
+    - python
+  run:
+    - ipywidgets
+    - python
+
+test:
+  imports:
+    - ipyfilechooser
+
+about:
+  home: "https://github.com/crahan/ipyfilechooser"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
It's ok to manually package the `license_file` as you did, but in future releases and updates to the feedstock, it would be desirable to include the license with the source distribution. If you could link to the following issue I just created as a comment here in the `meta.yaml` as a reminder in case the license file is available in future release's source distribution, so that the manually package license can be removed. 

https://github.com/crahan/ipyfilechooser/issues/20

Reply:
What field in meta.yaml should the issue link be added to?

Reply:
You can just add the link in a comment above the `license_file` field.

Reply:
Link has been added.

Reply:
It looks like using a URL instead of the LICENSE file name in the `license_file` field breaks the checks.

Reply:
Sorry for the confustion. What you want is to keep the manually packaged license file in this release and put the link as a comment above that field. So something like:

```
   # <url to issue>
   license_file: LICENSE
```

Reply:
Oh, I completely misread that! So sorry, I've corrected the issue now.